cartridges/app_storefront_pdp_extend/cartridge/client/default/js/product/detail.js

```diff
@@ -0,0 +1,62 @@
+var base = require("base/product/detail");
+
+function triggerChange(product, $productContainer) {
+    if (product.available) {
+        $productContainer.find("#addToCartButton").removeClass('d-none')
+        $productContainer.find("#phoneForm").addClass('d-none')
+    } else {
+        $productContainer.find("#addToCartButton").addClass('d-none')
+        $productContainer.find("#phoneForm").removeClass('d-none');
+    }
+}
+
+base.updateAvailability = function () {
```

Comment:
Please create one new exported function like: base.notify, which to define callback functions for the two custom events and to add there the logic only for the notify

Reply:
This is not resolved yet I believe